From a84abc0caba660444d8210107da4c03c03150cd4 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Wed, 24 May 2023 11:18:18 +0100
Subject: [PATCH] TODO: remove fixed item

(cherry picked from commit f19b62756071cd6fc28b800062f591a3af88fe6a)

Related: RHEL-16952
---
 TODO | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/TODO b/TODO
index c512bedb92..abe8faf5e8 100644
--- a/TODO
+++ b/TODO
@@ -249,12 +249,6 @@ Features:
   parametrization, if needed. This matches our usual rule that admin config
   should win over vendor defaults.
 
-* sd-stub: optionally allow users to configure manual kernel command line even
-  in SecureBoot by authenticating it via shim's APIs, integrating with MOK and
-  similar: instead of authenticating just PE code shim should be capable of
-  authenticating any kind of data for us, including files containing kernel
-  command lines.
-
 * write a "search path" spec, that documents the prefixes to search in
   (i.e. the usual /etc/, /run/, /usr/lib/ dance, potentially /usr/etc/), how to
   sort found entries, how masking works and overriding.
